src/store/compression_brotli.rs

```diff
@@ -10,12 +10,12 @@ pub fn compress(uncompressed: &[u8], compressed: &mut Vec<u8>) -> io::Result<()>
     params.quality = 5;
 
     compressed.clear();
-    brotli::BrotliCompress(uncompressed, compressed, &params)?;
+    brotli::BrotliCompress(&mut Cursor::new(uncompressed), compressed, &params)?;
```

Comment:
Is `Cursor` required here?

Reply:
Yep, first arg shall be mutable Read object. I dunno what is another way to convert immutable `&[u8]` to mutable Read. `&uncompressed[..]` is immutable, and `&mut uncompressed[..]` is not allowed due to immutability of `uncompressed`.